go-controller/pkg/ovn/service.go

```diff
@@ -169,14 +171,33 @@ func (ovn *Controller) syncServices(services []interface{}) {
 						if hasEps {
 							klog.Infof("Service Sync: Removing OVN stale reject ACL: %s", name)
 							ovn.removeACLFromPortGroup(lb, uuid)
+							var foundSwitches []string
+							// For upgrade from a non-port group Reject ACL implementation
```

Comment:
should there be a note that we can eventually remove this code, like there used to be on `removeACLFromNodeSwitches`?

Reply:
yeah I can add a comment here. Note, I removed that comment from the removeACLFromNodeSwitches as we need it for ACLs on the external switches now and in the foreseeable future.